src/screens/GovernanceCall/GovernanceCallBanner.js

```diff
@@ -0,0 +1,191 @@
+import { mapFromDocumentDataToString } from 'utils/prismic';
+import { formatDate } from 'utils/date';
+
+// Services
+import * as Prismic from 'services/prismic';
+
+// assets
+const smartWalletIcon = require('assets/icons/smart-wallet-migrate.png');
+
+const TYPE_GOVERNANCE_CALL = 'governance-calls';
+const DATE_FORMAT = 'dd.MM';
+const TIME_WITH_TIMEZONE_FORMAT = 'HH:mm OOO';
+
+const GovernanceCallBanner = () => {
+  const { t } = useTranslationWithPrefix('governanceCall');
+  const [title, setTitle] = React.useState(t('banner.title'));
+  const [description, setDescription] = React.useState('');
+  const [scheduledCallTime, setScheduledCallTime] = React.useState(0);
+  const [governanceCallLink, setGovernanceCallLink] = React.useState('');
+  const [showPermanently, setShowPermanently] = React.useState(false);
+
+  React.useEffect(() => {
+    async function fetchGovernanceCallData() {
+      try {
+        const data = await Prismic.queryDocumentsByType(TYPE_GOVERNANCE_CALL);
+        data?.results?.map((governanceCallData) => {
+          const bannerTitleContent = [];
+          mapFromDocumentDataToString(governanceCallData?.data?.title, bannerTitleContent);
+          setTitle(bannerTitleContent);
+          const bannerDescriptionContent = [];
+          mapFromDocumentDataToString(governanceCallData?.data?.description, bannerDescriptionContent);
+          setDescription(bannerDescriptionContent);
+          /* eslint-disable camelcase */
+          setScheduledCallTime(governanceCallData?.data?.scheduled_for);
+          setGovernanceCallLink(governanceCallData?.data?.link?.url);
+          setShowPermanently(governanceCallData?.data?.show_permanently);
+        });
+      } catch (error) {
+        reportErrorLog('Prismic content fetch failed', { error });
+      }
+    }
+    fetchGovernanceCallData();
+  }, [scheduledCallTime]);
+
+  const eventEndTime = addHours(new Date(scheduledCallTime), 1);
+  const formattedDate = formatDate(new Date(scheduledCallTime), DATE_FORMAT);
+  const formattedTimeWithTimezone = formatDate(new Date(scheduledCallTime), TIME_WITH_TIMEZONE_FORMAT);
+  const getTotalDays = differenceInDays(new Date(scheduledCallTime), Date.now());
+  const getTotalHours = differenceInHours(new Date(scheduledCallTime), Date.now());
+  const getTotalMinutes = differenceInMinutes(new Date(scheduledCallTime), Date.now());
+  const eventISLive = !isBefore(eventEndTime, Date.now()) && getTotalMinutes <= 0;
+
+  const isBannerShow =
+    (showPermanently && isAfter(eventEndTime, Date.now())) || (getTotalHours <= 24 && getTotalHours >= 0);
+
+  if (!isBannerShow) return null;
+
+  const showRequestModal = () => {
+    Linking.openURL(governanceCallLink);
+  };
+
+  const calculateRemainingTime = () => {
+    if (getTotalDays > 0) {
+      return `${getTotalDays}d ${getTotalHours}h`;
+    } else if (getTotalHours > 0) {
+      return `${getTotalHours}h`;
+    }
+    return `${getTotalMinutes}m`;
```

Comment:
How about seconds? It will show `0 minutes` now which doesn't tell anything :( Also I think it should re-render every second to show countdown, wdyt @IAmKio ?

Reply:
Or nevermind... Anyone can still join <1min so I think we're good, but still wdyt about countdown?

Reply:
Yeah we do need at least a re-render every minute to show the component actually counting down @divyaBacancy

Reply:
https://user-images.githubusercontent.com/66996635/134019907-df29f346-b45d-4570-9206-17fdc5735506.mov